010_Intro/15_API.asciidoc

```diff
@@ -42,7 +42,7 @@ seen, you can even talk to Elasticsearch from the command line by using the
 NOTE: Elasticsearch provides official clients((("clients", "other than Java"))) for several languages--Groovy,
 JavaScript, .NET, PHP, Perl, Python, and Ruby--and there are numerous
 community-provided clients and integrations, all of which can be found in the
-http://www.elasticsearch.org/guide/[Guide].
+http://www.elastic.co/guide/[Elastic documentation].
```

Comment:
Link to https://www.elastic.co/guide/en/elasticsearch/client/index.html instead?